modules/sdk-coin-avaxp/src/lib/atomicTransactionBuilder.ts

```diff
@@ -0,0 +1,192 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { BN, Buffer as BufferAvax } from 'avalanche';
+import utils from './utils';
+import { TransactionBuilder } from './transactionBuilder';
+import {
+  SECPTransferInput,
+  SECPTransferOutput,
+  SelectCredentialClass,
+  TransferableInput,
+  TransferableOutput,
+} from 'avalanche/dist/apis/platformvm';
+import { Credential } from 'avalanche/dist/common';
+import { BuildTransactionError } from '@bitgo/sdk-core';
+import { DecodedUtxoObj } from './iface';
+
+/**
+ * Atomic transaction are the cross chain transaction.
+ */
+export abstract class AtomicTransactionBuilder extends TransactionBuilder {
+  private static _cChainId: BufferAvax = utils.cb58Decode('yH8D7ThNJkxmtkuv2jgBa4P1Rn3Qpr4pPr7QYNfcdoS6k6HWp');
+  protected _externalChainId: BufferAvax = AtomicTransactionBuilder._cChainId;
+
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+    this.transaction._fee = new BN(this.transaction._network.txFee);
+  }
+
+  /**
+   * The external chain is outside cross chain, where the internal is the one set in coinConfig.network.
+   * The external chain id could be source or destiny chain depend of the type transaction.
+   *
+   * @param chainId
```

Comment:
nit add type and description: 
```suggestion
   * @param {string} chainId - id of the external chain
```